.nvmrc

```diff
@@ -0,0 +1 @@
+lts/fermium
```

Comment:
Fermium zit in maintenance fase, kunnen we gallium gebruiken of is er een specifieke reden?

Reply:
Nee geen specifieke reden, just muscle memory 😄 
Aangepast naar `lts/gallium`